static/app/components/button.spec.tsx

```diff
@@ -39,3 +27,17 @@ describe('Button', function () {
     expect(spy).not.toHaveBeenCalled();
   });
 });
+
+describe('LinkButton', function () {
+  it('renders react-router link', function () {
+    render(<LinkButton to="/some/route">Router Link</LinkButton>);
+  });
+
+  it('renders normal link', function () {
+    render(<LinkButton href="/some/relative/url">Normal Link</LinkButton>);
+  });
+
+  it('renders disabled normal link', function () {
+    render(<LinkButton href="/some/relative/url">Normal Link</LinkButton>);
```

Comment:
should this have the `disabled` prop?

Reply:
agree with @michellewzhang ... this should have the disabled prop and it would be nice to test if the button is unclickable like here https://github.com/getsentry/sentry/blob/af68d66484840ce61c48059bdf4764eb6b5efb52/static/app/components/button.spec.tsx#L30-L39